packages/spritesheet/src/SpritesheetLoader.ts

```diff
@@ -56,19 +56,23 @@ export class SpritesheetLoader
         // Check and add the multi atlas
         // Heavily influenced and based on https://github.com/rocket-ua/pixi-tps-loader/blob/master/src/ResourceLoader.js
         // eslint-disable-next-line camelcase
-        if (resource.data?.meta?.related_multi_packs)
+        if (resource.data?.meta?.related_multi_packs && Array.isArray(resource.data?.meta?.related_multi_packs))
```

Comment:
You don't need the first part of this of statement, can just go straight to the isArray part, as ?. will return undefined and fail the isArray check if it doesn't exist

Reply:
This was covered in the less verbose refactor from @bigtimebuddy 👍